src/libsyncengine/syncpal/syncpalworker.cpp

```diff
@@ -69,7 +70,15 @@ void SyncPalWorker::execute() {
     }
     perfMonitor.stop();
 
+    if (!sleepUntilStartDelay()) {
+        // Exit
```

Comment:
```suggestion
        // Stop was asked
```
Ideally, the name of the function would make comment useless.